app/assets/translations/appOfflineData/getDataToStore.ts

```diff
@@ -146,66 +146,136 @@ const getAllDataToStore = async (languageCode: string, dispatch: any, prevPage:
 export const getAllDataOnRetryToStore = async (apiEndpoint: string, languageCode: string, dispatch: any, _prevPage: string, activeChild?: any): Promise<any> => {
     // return new Promise(async (resolve) => {
     let Entity: any;
-    if (apiEndpoint == appConfig.basicPages) {
-        await getDataToStore(languageCode, dispatch, BasicPagesSchema, Entity as BasicPagesEntity, basicPagesData, setAllTermsData);
+    if (apiEndpoint == appConfig.apiConfig.basicPages) {
+        try {
+            await getDataToStore(languageCode, dispatch, BasicPagesSchema, Entity as BasicPagesEntity, basicPagesData, setAllTermsData);
+        } catch (error) {
+            console.log('error', error)
```

Comment:
Use console.error instead of console.log for better debugging.
`console.error('Error fetching basic pages data:', error);`